src/main/java/sirius/web/data/XLSProcessor.java

```diff
@@ -153,4 +184,19 @@ private Object extractNumericValue(Cell cell) {
             return val;
         }
     }
+
+    /**
+     * Specifies a list of names for sheets that are to be processed.
+     *
+     * @param sheetNames a list of sheet names to process
+     * @return the processor itself for fluent method calls
+     */
+    public XLSProcessor withSheetNames(List<String> sheetNames) {
```

Comment:
do we need this, if we pass the sheet along in the sheet to the RowProcessor?

Reply:
no, we don't